src/service/video-manager-impl.js

```diff
@@ -355,6 +355,36 @@ export class VideoManager {
   }
 
   /**
+   * Returns the entry in the video manager corresponding to the element
+   * provided
+   *
+   * @param {!AmpElement} element
+   * @return {VideoEntry} entry
+   * @private
+   */
+  getEntryForElement_(element) {
+    for (let i = 0; i < this.entries_.length; i++) {
+      const entry = this.entries_[i];
+      if (entry.video.element === element) {
+        return entry;
+      }
+    }
+    dev().assert(false, 'video is not registered to this video manager');
```

Comment:
I agree that `dev().error()` is more appropriate than `dev().assert()`. Here I was mirroring the behavior of [getEntryForVideo_](https://github.com/cvializ/amphtml/blob/2f019dfa358f6227d6b0e1b568d42772fac984df/src/service/video-manager-impl.js#L353). Should I change it there too?